about.html

```diff
@@ -0,0 +1,136 @@
+ <a class="pop-window" href="#footer"> Join</a>   
+ <a class="pop-window" href="#footer"> Sponsors</a>  
+ <a class="pop-window" href="#footer"> News</a>  
+</div>
+
+<!-- HAMBURGER-WINDOW end-->
+  <!-- SECTION 1 -->
+ <section id="section-about">
+  <div id="about-div">
+    <div class="social-media">
+      <i class="fa-brands fa-square-facebook fa-2x"></i>
+      <i class="fa-brands fa-square-twitter fa-2x"></i>
+    </div>
+    <nav id="navbar">
+      <div id="nav-content"> 
+        <a id="ham-link" >
+        <img id="ham-btn" src="/bg-img/Hamburger_icon.svg.png" alt="hamburger-icon">
+        </a>
+        <a id="logo-mobile"> <img id="logo-btn-show" src="/bg-img/logo.png" alt="logo-icon"> </a>
+        <a id="logo-link" href="/index.html">
+          <img id="logo-btn" src="/bg-img/logo.png" alt="logo-icon">
+          </a>
+        <div id="nav-pages">
+          <a class="pop-window" href="/about.html">About</a>
+          <a class="pop-window" href="#about">Program</a>
+          <a class="pop-window" href="#footer"> Join</a>   
+          <a class="pop-window" href="#footer"> Sponsors</a>  
+          <a class="pop-window" href="#footer"> News</a>  
+        </div>
+        <div id="contact"> 
+          <button id="contact-btn" > Contact </button>
+        </div>
+      </div>
+    </nav>
+   <div id="about-bg"> 
+    <div class="title-about">
+     <h1> GET READY FOR THE ULTIMATE FESTIVAL WINTER 2023 </h1>
+    </div>
+    <div class="paragraph" id="about-p"> Lolla takes over historic Grant Park in Downtown Chicago, alongside the beautiful Chicago skyline and the shores of Lake Michigan. Take advantage of interactive areas 
+     with giveaways, merch and more!</div>
+    <div class="date" >SAVE THE DATE! MARCH 2-5-2023
+     <p>@ Chicago, Illinois, USA.</p>
+    </div>
+   </div>
+  </div>
+ </section>
+
+ <!-- SECTION 2 -->
+
+ <section id="section-2">
+  <div id="about-bg-2"> 
+   <div class="title-about-2">
+    <h1> ULTRA-PALLOZA LOGO <div id="program-separator-line" class="guide-bar"></div></h1>
+   </div>
+   <div class="paragraph" id="about-p">
+    <p>
+     The ULTRA-PALLOZA logo was made by  street artist BAKSY in 2022
+    </p> 
+   </div>
+   <div id="about-img-2">
+    <img src="/bg-img/Captura de pantalla 2022-10-18 125031.png" alt="festival-img">
+   </div>
+  </div>
+ </section>
+
+ <!-- SECTION-3 -->
+
+ <section id="section-3">
+  <div id="about-bg-3"> 
+   <div class="title-about-2">
+    <h1> SEE THE PREVIOUS EDITON OF THE FESTIVAL <div id="program-separator-line" class="guide-bar"></div></h1>
+   </div>
+   <div class="paragraph" id="about-p"> ULTRA PALLOZA is a place to relax and meet new friends, listen to some cool indie bands and chill</div>
+   <div id="about-img-3">
+    <div id="red-img"> 
+     <h1> 2021</h1>
+     <h3>ULTRA-PALLOZA Buenos aires</h3>
+    </div>
+    <div id="red-img-2"> 
+     <h1> 2020</h1>
+     <h3> ULTRA-PALOZZA Quito</h3>
+    </div>
+   </div>
```

Comment:
- [ ] I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).